build.gradle

```diff
@@ -500,6 +500,14 @@ project('spring-integration-mail') {
 	[compileJava,compileTestJava]*.options*.compilerArgs = ["${xLintArg},-path"]
 }
 
+project('spring-integration-mock') {
```

Comment:
I wonder if we should rename the current `spring-integration-test` to `spring-integration-test-support` and call this one `spring-integration-test` - after all, it's not only about mocks.